src/MIDebugEngine/Engine.Impl/DebuggedThread.cs

```diff
@@ -313,7 +313,19 @@ private async Task<List<ThreadContext>> WalkStack(DebuggedThread thread)
         private ThreadContext CreateContext(TupleValue frame)
         {
             ulong? pc = frame.TryFindAddr("addr");
-            MITextPosition textPosition = MITextPosition.TryParse(this._debugger, frame);
+
+            // don't report source line info for modules marked as IgnoreSource
+            DebuggedModule module = null;
```

Comment:
nit: I had to read this code a few times to understand what it was doing. How about --

```
            // don't report source line info for modules marked as IgnoreSource
            bool ignoreSource = false;
            if (pc != null)
            {
                module = _debugger.FindModule(pc.Value);
                if (module != null && module.IgnoreSource)
                {
                    ignoreSource = true;
                }
            }
            MITextPosition textPosition = !ignoreSource ? MITextPosition.TryParse(this._debugger, frame) : null;
```